.ci/setup_env_rhel.sh

```diff
@@ -11,11 +11,14 @@ cidir=$(dirname "$0")
 source "/etc/os-release" || "source /usr/lib/os-release"
 source "${cidir}/lib.sh"
 
-echo "Get repo for perl-IPC-Run"
-sudo -E yum-config-manager --enable rhui-rhel-7-server-rhui-optional-rpms
+if [ "$(echo "${VERSION_ID} < 8.0" | bc -q)" == "1" ]; then
+	echo "Get repo for perl-IPC-Run"
+	sudo -E yum-config-manager --enable rhui-rhel-7-server-rhui-optional-rpms
+fi
 
 echo "Add epel repository"
-epel_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+get_version=$(echo $VERSION_ID | cut -d '.' -f1)
```

Comment:
variable name doesn't make sense to me, what about renaming it to `major_version`?